pages/get-involved.tsx

```diff
@@ -134,6 +141,61 @@ function PerfectFit() {
   );
 }
 
+function TestimonialSection() {
+  const theme = useTheme();
+  const isMdOrLarger = useMediaQuery(theme.breakpoints.up('md'));
+  const settings = {
+    dots: true,
+    infinite: true,
+    // autoplaySpeed: 5000,
```

Comment:
Do we need these key/vals in comments here?

Reply:
Yea I am going to reach out to Dan to see if we want it to autoplay or not.

Reply:
removing autoplay